lib/simulation/simulated_peripheral.dart

```diff
@@ -157,4 +157,9 @@ abstract class SimulatedPeripheral {
   }
 
   Future<int> rssi() async => scanInfo.rssi;
+
+  Future<int> requestMtu({int requestedMtu}) async {
+    mtu = requestedMtu ?? mtu;
```

Comment:
requestedMtu shouldn't be optional and the requestedMtu value should be limited to 23 on lower bound and 512 on upper bound.

Reply:
but there is no sense to send MTU value, that is why it is optional,
how does the lib behave when somen request 513 or 22?

Reply:
I mean the SimulatedPeripheral should not got below 23 or above 512, since these are standard defined in Bluetooth Low Energy specification. The library itself just pass it to the peripheral, which does the negotiating on its own.

What do you mean there's no sense in sending a value we're requesting to be set?

Reply:
It's also weird for me to allow user to pass null MTU value. What is the sense of requesting mtu then?

Reply:
on iOS there is not any sense but a user can call the method